nil/internal/collate/syncer.go

```diff
@@ -358,11 +358,9 @@ func (s *Syncer) GenerateZerostate(ctx context.Context) error {
 		return nil
 	}
 
-	if len(s.config.BlockGeneratorParams.MainKeysPath) != 0 && s.config.ShardId == types.BaseShardId {
```

Comment:
At least for now you don't need to load keys for everything except base shard. Why do you need this change?

Reply:
Because now the main purpose of this keys - to make validator execute some transactions using this case. This means, that if I run only node with shard0+2, I'm also want to use MainPrivateKey to execute transactions.

Reply:
That's pretty strange. We use this key only as a key for main smart account. It's not related to validators anyhow.

Reply:
However, seems I got all you meant

Reply:
Okay. But who can use this key? 
If we load it only for baseShard only validator on BaseShard can use it. That’s also strange behavior.